assets/js/components/search-list-control/style.scss

```diff
@@ -49,13 +49,24 @@
 	}
 
 	.woocommerce-search-list__item {
+		display: flex;
+		align-items: center;
 		margin-bottom: 0;
 		padding: $gap;
 		background: $white;
 		// !important to keep the border around on hover
 		border-bottom: 1px solid $core-grey-light-500 !important;
 		color: $core-grey-dark-500;
 
+		svg {
```

Comment:
It may not be immediately obvious what this is for. Would it be possible to use a descriptive class selector instead?